src/Assets/Lists/allEvents.js

```diff
@@ -58,6 +58,25 @@ export const allEvents = [
         link: "https://hackstudentlife.ca/",
         highlightEvent: true,
     },
+    {
+        EventName: "Technical Interview Night",
+        displayDate: "October 3, 2024",
+        date: "2024/10/03",
+        highlightEvent: false,
+    },
+    {
+        EventName: "NewHacks 2024",
```

Comment:
For this, delete the _NewHacks 2024.jpg_ under /src/Assets/Images/events, and use the one from 2023, we don't need a duplicate one.